test/drivers/utils.ts

```diff
@@ -63,6 +63,37 @@ export function testDriver(opts: TestOptions) {
     );
   });
 
+  it("getKeys with depth", async () => {
+    await ctx.storage.setItem("depth0:depth1:depth1_0", "test_data");
+    await ctx.storage.setItem("depth0:depth1:depth1_1", "test_data");
+    await ctx.storage.setItem("depth0:depth0_0", "test_data");
+    await ctx.storage.setItem("depth0:depth0_1", "test_data");
+    const depth0Keys = await ctx.storage.getKeys(undefined, { depth: 0 });
+    const depth1Keys = await ctx.storage.getKeys(undefined, { depth: 1 });
+    const depth2Keys = await ctx.storage.getKeys(undefined, { depth: 2 });
+
+    depth1Keys.sort();
+    depth2Keys.sort();
+
+    expect(depth0Keys).lengthOf(0);
```

Comment:
What would be the use case of `{ depth: 0 }` => [] you think?

IMO it could simply return top-level keys without `:`.

Reply:
it would return keys which have no delimiter right now

are you saying you'd consider "top level" as the first delimited level? e.g. `foo:bar`, is top level because `foo` is the "root"?

Reply:
Ah, then that's fine. We could perhaps add a `root` key to the test so that we test this behavior, it was a little misleading.